Angular/src/app/_services/data.service.ts

```diff
@@ -173,18 +174,58 @@ export class DataService {
       });
   }
 
-  getBandInformation(bandId: number) {
+  createCapability(param: any) {
     return axios
-      .get(getUniqueBandURL, {
+      .post(addCapabilityURL, param, { headers: this.getHeaders() })
+      .then(function (response) {
+        return response;
+      })
+      .catch(function (error) {
+        if (error.response) {
+          return {
+            error: error.response.data.Message,
+            statsuCode: error.response.statusCode
```

Comment:
Should be spelled `statusCode` here. Has this feature been tested yet? It should be throwing an error here